api/main.py

```diff
@@ -122,3 +122,67 @@ def root():
     except KeyError as e:
         print("KeyError: ", e)
         raise HTTPException(status_code=400)
+
+
+@app.get("/menus/{language}")
+async def get_items_for_menu(language: str):
+    try:
+        db = get_db()
+        menu_query_result = db.AQLQuery(
+            query=query_items_for_menu,
+            bindVars={"language": get_language_from_languagename(language)}
```

Comment:
question: is the transforming function necessary? Can't we just pass the desired language name in the request?

Reply:
Then we have to transform the language in the frontend before it is passed in. 
Or we should make the entire frontend just using the abbreviated languages and not the full language.